python/servo/devenv_commands.py

```diff
@@ -108,15 +109,30 @@ def cargo_fix(self, params, **kwargs):
 
     @Command("clippy", description='Run "cargo clippy"', category="devenv")
     @CommandArgument("params", default=None, nargs="...", help="Command-line arguments to be passed through to clippy")
+    @CommandArgument("--report-ci", "-rci", default=False, action="store_true", help="Put the lint result on the file")
```

Comment:
```suggestion
    @CommandArgument("--report-ci", "-rci", default=False, action="store_true", help="Emit the clippy warnings in  the Github Worfklow command format")
```

Reply:
I'd probably also just remove the shorthand `-rci` option as that is not useful for developers.